### DISCUSSION TITLE
How will osu!lazer deal with external tools like Storybrew? Will there be an API for external mapping tools?

### DISCUSSION THREAD

**Discussion by CenTdemeern1** [ppy/osu#18037] [Q&A]
I personally use [Storybrew](https://github.com/Damnae/storybrew/) for making storyboards, and since lazer stores everything in Realm, I feel like this would cause some major trouble (specifically because you can't access .osu and .osb files directly anymore).
Will there be some kind of interface to mitigate this issue and allow for external editing tools?

If lazer is going to be a full-on replacement for stable, "with no reason to go back to stable", this will need to be mitigated. Because this sounds like a pretty good reason to use stable over lazer for mapping (especially since I assume the lazer design mode isn't going to allow for C# programming).

**Comment by frenzibyte** [comment]
Probably best to couple this with https://github.com/ppy/osu/discussions/16570

**Comment by bdach** [comment]
also #12029
